### PR TITLE
add attribute columns to migrations

### PR DESCRIPTION
**Pull request by steveklabnik**
r? @ashleygwilliams

### CODE REVIEW

File: migrations/3_create_customers.js [hunk 1]
```diff
@@ -3,6 +3,16 @@ exports.up = function (knex) {
...
       t.integer('artist_id').notNullable().references('id').inTable('artists');
+      t.string('name');
+      t.string('phone');
+      t.string('email');
+      t.dateTime('date_on_waiting_list');
+      t.integer('status');
+      t.boolean('local');
+      t.string('location');
+      t.string('credit');
+      t.text('want');
```
Comment (ashleygwilliams) [inline migrations/3_create_customers.js]: going on record as saying this is a weird name

File: migrations/4_create_appointments.js [hunk 2]
```diff
@@ -4,6 +4,8 @@ exports.up = function (knex) {
       t.increments('id');
       t.integer('artist_id').notNullable().references('id').inTable('artists');
       t.integer('customer_id').notNullable().references('id').inTable('customers');
+      t.datetime('date_scheduled');
```
Comment (ashleygwilliams) [inline migrations/4_create_appointments.js]: let's make this camel for consistency's sake. even though it seems to work... http://knexjs.org/#Schema-dateTime

File: package.json [hunk 3]
```diff
@@ -29,8 +29,7 @@
     "express": "^4.10.6",
     "express-routebuilder": "^2.1.0",
     "knex": "^0.8.0",
-    "pg": "^4.4.0",
-    "sqlite3": "^3.0.4"
```
Comment (ashleygwilliams) [inline package.json]: lol oops on my part. thanks.

**Comment by ashleygwilliams** [comment]
please go back through and add uniqueness and nonNullable constraints :)

File: migrations/3_create_customers.js (hunk 1 above)
Comment (steveklabnik) [inline migrations/3_create_customers.js]: legacy...

File: package.json (hunk 3 above)
Comment (steveklabnik) [inline package.json]: :heart:

**Comment by steveklabnik** [comment]
ready for another r?

**Comment by ashleygwilliams** [comment]
looks good to me. r+ :dancers:

**Pull request merged**